src/ui.form-calendar/index.vue

```diff
@@ -231,6 +231,14 @@ const props = defineProps({
     default: getDefault(defaultConfig, UCalendar).dateFormat,
   },
 
+  /**
+   * Same as date format, but used when timepicker is enabled.
+   */
+  timeFormat: {
+    type: String,
+    default: getDefault(defaultConfig, UCalendar).timeFormat,
+  },
+
   /**
    * User friendly date format.
    */
```

Comment:
what about renaming `userFormat` into `userDateFormat` and add separate `userDateTimeFormat`

Reply:
User format is independent of timepicker. I don't see any reason for additional props, this will only increase complexity of the component.

Reply:
Let's discuss it in zoom later.